src/transport/MMCollisionInt.h

```diff
@@ -19,6 +19,17 @@ namespace Cantera
  * This class provides functions that interpolate the tabulated collision
  * integrals in Monchick and Mason @cite monchick1961.
  *
+ * The collision integrals computed by Monchick and Mason use the Stockmayer
+ * potential, which models a polar molecule as a spherical potential with a
+ * point dipole at the center).
+ *
+ * @f[
+ *    \phi(r) = 4\epsilon \left[ \left(\frac{\sigma}{r}\right)^{12} - \left(\frac{\sigma}{r}\right)^6 + \delta \left(\frac{\sigma}{r}\right)^3 \right]
+ * @f]
```

Comment:
Adding the equation is nice, but it would help to define what the variables are.